src/components/ContactsList/ContactsSubList.jsx

```diff
@@ -5,10 +5,15 @@ import ContactRow from './ContactRow'
 
 const ContactsSubList = ({ contacts }) => {
   return (
-    <ol className="sublist-contact">
-      {contacts.map(contact => (
+    <ol className="u-m-0 u-pl-0 u-nolist">
+      {contacts.map((contact, index) => (
         <li key={`contact-${contact._id}`}>
```

Comment:
Is the List / ListItem not usable now ? I think ContactRows should render ListItem and the border-bottom: 0 should be disabled via divider={false}.